scripts/criu-ns

```diff
@@ -124,6 +126,18 @@ def wrap_restore():
     criu_pid = os.fork()
     if criu_pid == 0:
         os.setsid()
+        '''
+        Set stdin tty to be a controlling tty of our new session, this is
+        required by --shell-job option, as for it CRIU would try to set a
+        process group of restored root task to be a foreground group on the
+        terminal.
+        '''
```

Comment:
The [convention](https://peps.python.org/pep-0257/) in Python is to always use `"""` (triple double quotes) for docstrings.
For [block comments](https://peps.python.org/pep-0008/#block-comments), each line of starts with a `#` and a single space.

Reply:
Thanks, will fix!